views/blocks/background-color.twig

```diff
@@ -10,6 +10,11 @@
     {% set classes = classes|merge(['l-full-width']) %}
 {% endif %}
 
+{% if attributes.float %}
+    {% set classes = classes|merge(['u-align--'~attributes.float]) %}
+    {% set classes = classes|merge(['style="max-width:33%;']) %}    
```

Comment:
should be set as styles variable, not classes.

Reply:
My bad, sorry. I've pushed a fix for that. I looked at it visually and it appeared to take just 33% of the space, but that was because my content wasn't super long.